filer/templates/admin/filer/breadcrumbs.html

```diff
@@ -11,7 +11,7 @@
             {% if ancestor_folder.label %}
                 {{ ancestor_folder.label }}
             {% else %}
-                {% if ancestor_folder.name != "unfiled files" %}
+                {% if ancestor_folder.name != _("Unsorted Uploads") %}
```

Comment:
this is an incorrect template syntax. there is no ``_`` function in the templates.
aslo I'm not sure that this will work with translated string. What will happen if current language is German, and the string ``Unsorted Uploads`` is translated (has a german translation instead of falling back to English?